apps/web/composables/useCategoryTemplate/homepageTemplateDataDe.json

```
@@ -185,25 +188,67 @@
     }
   },
   {
-      "name": "NewsletterSubscribe",
-      "type": "content",
-      "meta": {
-        "uuid": "0f1e2d3c-4b5a-4c7d-8e9f-1a2b3c4d5e6f"
+    "name": "NewsletterSubscribe",
+    "type": "content",
+    "meta": {
+      "uuid": "0f1e2d3c-4b5a-4c7d-8e9f-1a2b3c4d5e6f",
+      "isGlobalTemplate": false
+    },
+    "content": {
+      "text": {
+        "bgColor": "#f5f5f5",
+        "title": "Abonnieren Sie unseren Newsletter",
+        "htmlDescription": "Informieren Sie sich über bevorstehende Aktionen und Veranstaltungen. Erhalten Sie Geschenke und Sonderangebote!"
+      },
+      "input": {
+        "displayNameInput": true,
+        "nameIsRequired": false
       },
-      "content": {
+      "button": {
+        "label": "Newsletter abonnieren"
+      }
+    }
+  },
+  {
+    "name": "Footer",
+    "type": "content",
+    "meta": {
+      "uuid": "4f7e2b1a-9c8d-4e6f-b3a2-1d0c5b4a3e2f",
+      "isGlobalTemplate": true
+    },
+    "content": {
+      "column1": {
         "text": {
-          "bgColor": "#f5f5f5",
-          "title": "Abonnieren Sie unseren Newsletter",
-          "htmlDescription": "Informieren Sie sich über bevorstehende Aktionen und Veranstaltungen. Erhalten Sie Geschenke und Sonderangebote!"
-        },
-        "input": {
-          "displayNameInput": true,
-          "nameIsRequired": false
-        },
-        "button": {
-          "label": "Newsletter abonnieren"
+          "title": "Rechtliches"
         }
+      },
+      "column2": {
+        "text": {
+          "title": "Kontakt",
+          "showLinkToContact": false,
+          "htmlDescription": "<ul><li><a href='https://plentyshop.plentymarkets.com/contact'></a></li></ul>"
```

```suggestion
          "htmlDescription": "<ul><li><a href='https://plentyshop.plentymarkets.com/de/contact'></a></li></ul>"
```
Also, **question**: Shouldn't we just add the slug here? So `/de/contact`?